doc/concepts/simulation.rst

```diff
@@ -2,28 +2,33 @@
 
 Simulations
 ===========
-A simulation is the executable form of a model and is used to interact with and monitor the model state. In the simulation the neuron model is initiated and the spike exchange and the integration for each cell group are scheduled.
+A simulation is the executable form of a model and is used to interact with and monitor the model state.
+In the simulation the neuron model is initiated and the spike exchange and the integration for each cell
+group are scheduled.
 
 From recipe to simulation
 -------------------------
 
 To build a simulation the following are needed:
 
-    * A recipe that describes the cells and connections in the model.
-    * A context used to execute the simulation.
+    * A :ref:`recipe <modelrecipe>` that describes the cells and connections in the model.
+    * A :ref:`domain decomposition <modeldomdec>` that describes the distribution of the
+      model over the local and distributed :ref:`hardware resources <modelhardware>`.
+    * An :ref:`execution context <modelcontext>` used to execute the simulation.
 
-The workflow to build a simulation is to first generate a domain decomposition that describes the distribution of the model over the local and distributed hardware resources (see :ref:`modeldomdec`), then build the simulation from the recipe, the domain decomposition and the execution context. Optionally experimental inputs  that can change between model runs, such as external spike trains, can be injected.
-
-The recipe describes the model, the domain decomposition describes how the cells in the model are assigned to hardware resources and the context is used to execute the simulation.
+Optionally, experimental inputs that can change between model runs, such as external spike trains, can be injected.
 
 Simulation execution and interaction
 ------------------------------------
 
 Simulations provide an interface for executing and interacting with the model:
 
-    * The simulation is executed/ *run* by advancing the model state from the current simulation time to another with maximum time step size.
+    * The simulation is executed/ *run* by advancing the model state from the current simulation time to another
```

Comment:
The `executed/ *run*` bit does not make much sense to me, is it correct grammatically?

Reply:
So I think the intention is to signify that the execution of a simulation is referred to as running the simulation in Arbor. The past participle of run is still run, so it's grammatically correct.

Reply:
Does the space between `executed/ *run*` disappear when rendered?

Reply:
No, I'll remove the space.